app/assets/stylesheets/modules/spotlight_overrides.scss

```diff
@@ -195,3 +195,8 @@ h5,
 .img-thumbnail > a > img {
   @include thumbnail-image-border;
 }
+
+.tags .nav-link {
+  margin-bottom: 0.5rem;
+  padding: 0.33rem 0.66rem;
```

Comment:
Is this something that should be upstream or is this specific to our app?

Reply:
I talked with @ggeisler about this. His recommendation was that Spotlight should continue using the BS4 nav pills and this custom styling should reside in Exhibits. Exhibits uses a different approach to styling the nav pills than BS4 (pills are button looking vs BS4 only active looks like a button)

Reply:
![gotcha](https://media0.giphy.com/media/l2QDXm5xRQbUXHcvS/giphy.gif)

Reply:
![](https://media.giphy.com/media/uZsBBWQUcfnm8/giphy.gif)